Tests/scripts/validate_files_structure.py

```diff
@@ -514,7 +521,7 @@ def validate_added_files(added_files, integration_set, playbook_set, script_set,
     has_schema_problem = False
     for file_path in added_files:
         print "Validating {}".format(file_path)
-        if not validate_schema(file_path):
+        if not validate_schema(file_path) or validate_version(file_path):
```

Comment:
shouldn't it be `not validate_version(file_path)` ?

Reply:
nope it returns true if it has validation issue